invenio_communities/utils.py

```diff
@@ -109,3 +110,16 @@ def on_datastore_post_commit(sender, session):
             users = role.users.all()
             for user in users:
                 on_user_membership_change(Identity(user.id))
+
+
+def humanize_byte_size(size):
+    """Converts bytes to largest unit (e.g., KB, MB, GB)."""
+    old_rounding = getcontext().rounding
+    getcontext().rounding = ROUND_HALF_UP
+    s = Decimal(size)
+    for unit in ["B", "KB", "MB", "GB", "TB", "PB"]:
+        if s < 1000.0:
+            s = round(s, 2)
+            getcontext().rounding = old_rounding
+            return s, unit
+        s /= 1000.0
```

Comment:
I did not include a note about using a different library as a comment because I didn't want to overstep as a first time contributor. I get the feeling that regular contributors who have more experience will have opinions on this, so it might make more senes to discuss this in the PR.